Lib/fontTools/ttLib/tables/_c_m_a_p.py

```diff
@@ -38,6 +38,20 @@ def getcmap(self, platformID, platEncID):
 				return subtable
 		return None # not found
 
+	def getBestCmap(self, cmapPreferences=((3, 10), (3, 1), (0, 3))):
```

Comment:
just for symmetry, maybe you could also add plaftormID=0 (Unicode), platEncID=4 (non-BMP, Unicode full repertoire), since you also added (3, 10)

Reply:
In which order should it be in the default list?

Reply:
i'd say `(3, 10), (0, 4), (3, 1), (0, 3)`